packages/ppa-nest-api/src/planning-poker/planning-poker.gateway.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  WebSocketGateway,
+  SubscribeMessage,
+  MessageBody,
+} from '@nestjs/websockets';
+import { PlanningPokerService } from './planning-poker.service';
+
+@WebSocketGateway()
+export class PlanningPokerGateway {
+  constructor(private readonly planningPokerService: PlanningPokerService) {}
+
+  // @SubscribeMessage('createPlanningPoker')
```

Comment:
why leaving commented-out code?

Reply:
I didn't mean to commit this but this is work in progress...
I just don't want to remove it yet because I want to fully understand how this works.